web/app/js/components/NetworkGraph.jsx

```diff
@@ -0,0 +1,217 @@
+import _ from 'lodash';
+import ErrorBanner from './ErrorBanner.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withContext } from './util/AppContext.jsx';
+import * as d3 from 'd3';
+import 'whatwg-fetch';
+
+
+const defaultSvgWidth = 494;
+const defaultSvgHeight = 295;
+const margin = { top: 0, right: 0, bottom: 10, left: 0 };
+
+const simulation = d3.forceSimulation()
+  .force("link", d3.forceLink().id(function(d) { return d.id; }).distance(60))
```

Comment:
A little more distance and upping the node radius might be nice to make use of the graph's space

```
.force("link", d3.forceLink().id(function(d) { return d.id; }).distance(100))
```